src/murmur/Cert.cpp

```diff
@@ -116,6 +117,27 @@ void Server::initializeCert() {
 		qlCA = ql;
 	}
 
+#if defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
+	if (! dhparams.isEmpty()) {
+		QSslDiffieHellmanParameters qdhp = QSslDiffieHellmanParameters(dhparams);
+		if (qdhp.isValid()) {
+			qsdhpDHParams = qdhp;
+		} else {
+			QSet<QSslDiffieHellmanParameters::Error> errors = qsdhpDHParams.errors();
```

Comment:
Yeah. I think that would definitely be easier to work with. The errorString would definitely be convenient for our use-case too. As long as you can always fix the one error to reach the next one being able to see all of them at once is probably just complicating things for most applications.